docs/docs-content/glossary-all.md

```diff
@@ -68,6 +68,10 @@ fully encrypted using the tenant's unique encryption key.
 Cluster Mode enables you to create, deploy, and manage Kubernetes clusters and applications. In Cluster Mode, you can
 deploy Kubernetes clusters to public cloud providers, on-prem data centers, and on the edge.
 
+## Cluster Definition
+
+A Cluster Definition contain one or more cluster profiles and profile variables used in those profiles.
```

Comment:
Could we maybe add a bit more details 😅 
Maybe something that explains that it's a bundle of the two (cp and cp variables) and can be used with the EMC console to create clusters.

Reply:
agree with @karl-cardenas-coding  - the definition of "cluster definition" is important to make very clear, as it is, imo, a fairly difficult concept to understand.  afaik, it is a bundle of 1 or more CPs and with optionally associated cluster variables. I also don't think this explanation should only be in the glossary, or at least, it should be linked to from the EMC section. EMC [section](https://65e25712e2e4712afc8deff7--docs-spectrocloud.netlify.app/clusters/edge/edge-management-console/cluster-management/) atm referneces "cluster definition" but doesn't explain or link to what it is.